src/qt/optionsdialog.cpp

```diff
@@ -295,10 +295,22 @@ void OptionsDialog::on_resetButton_clicked()
 
 void OptionsDialog::on_openBitcoinConfButton_clicked()
 {
-    /* explain the purpose of the config file */
-    QMessageBox::information(this, tr("Configuration options"),
-        tr("The configuration file is used to specify advanced user options which override GUI settings. "
-           "Additionally, any command-line options will override this configuration file."));
+    QMessageBox config_msgbox;
+    config_msgbox.setIcon(QMessageBox::Information);
+    //: Window title text of pop-up box that allows you to open up your configuration file.
```

Comment:
nit:

Translator comments are kind of documentation. Could they be more formal, without "you" and "your"?

Reply:
Addressed in  0b869df